### PR TITLE
Comparison to old/new results after SNF/HNF changes

### PR DESCRIPTION
**Pull request by glwhart**
The "compare" code has been updated and used to compare the changes in output files since the SNF routine in symlib was changed. A few unit tests have been updated after the comparison code was used to show that the outputs (old and new) were equivalent, even though they are not identical.

### CODE REVIEW

File: src/labeling_related.f90 [hunk 1]
```diff
@@ -712,7 +711,15 @@ SUBROUTINE write_labelings(k,n,nD,parLabel,parDigit,HNFi,HNFlist,SNFlist,L,fixOp
        ! Now convert the base-10 number (labIndx) to the correct labeling
        if(conc_check) then
           labIndx = vsL(il)
+!del          write(*,'("labIndx,concVect,labeling")')
```
Comment (wsmorgan) [inline src/labeling_related.f90:714]: Do you want to delete the things you commented out with !del?

File: src/spacegroup.f90 [hunk 2]
```diff
@@ -0,0 +1,27 @@
+! stand alone driver to read in lattice vectors and atomic basis
```
Comment (wsmorgan) [inline src/spacegroup.f90:1]: This file is incomplete, i.e., it doesn't actually do anything and should probably be removed.

File: tests/messing_around/enumerate_periodic.in.txt [hunk 3]
```diff
@@ -0,0 +1,253 @@
+AMS_DATA
```
Comment (wsmorgan) [inline tests/messing_around/enumerate_periodic.in.txt]: Do we want to keep this directory as part of the repo?

**Review by wsmorgan — CHANGES_REQUESTED** [review]
3 things
1) !del comments delete them?
2) tests/messing_around directory probably not needed.
3) committed empty space group driver should probably be removed.

File: src/labeling_related.f90 (hunk 1 above)
Comment (glwhart) [reply src/labeling_related.f90:714]: Thanks Wiley, for catching these  things. I made some changes based on your feedback and a BUNCH more that are all cosmetic, like removing print statements, old comments, adding new comments that made it easier when debugging this time.

Take a look now and see what you think. I think my changes from this morning have been added to the PR.

File: src/spacegroup.f90 (hunk 2 above)
Comment (glwhart) [reply src/spacegroup.f90:1]: I'm going to leave it there until the "inactive" stuff is done. I can't remember how I was using it. Now that the "compare" stuff is done, I can try and finish up that.

File: tests/messing_around/enumerate_periodic.in.txt (hunk 3 above)
Comment (glwhart) [reply tests/messing_around/enumerate_periodic.in.txt]: Yes, for now. It's related to the "inactive site" stuff. But I renamed it. We'll probably delete it after the inactive stuff is done.

**Review by wsmorgan — APPROVED** [review]

**Pull request merged**